en_ewt-ud-train.conllu

```diff
@@ -164914,12 +164914,12 @@
 18	in	in	ADP	IN	_	19	case	19:case	_
 19	spite	spite	NOUN	NN	Number=Sing	26	obl	26:obl:in	_
 20	of	of	ADP	IN	_	19	nmod	19:nmod	_
-21	the	the	PRON	PRP	Typo=Yes	26	nsubj	26:nsubj	_
+21	the	the	PRON	PRP	Typo=Yes	26	nsubj	26:nsubj	CorrectForm=they
```

Comment:
pronoun features?

Reply:
token 21 should be
the	they	PRON	PRP	Case=Nom|Number=Plur|Person=3|PronType=Prs|Typo=Yes	26	nsubj	26:nsubj	CorrectForm=they